### ISSUE TITLE
Clarify relationship between SOSMAG and KOMPSAT

### ISSUE THREAD

**Issue by jameswilburlewis** [spedas/bleeding_edge#92]
Our naming conventions usually have load routines named after the mission, with specific instruments as keyword options.  Someone looking for KOMPSAT magnetometer data may not find it, because the load routine is named after the SOSMAG instrument.  We should clarify the relationship in our documentation, GUI control text, and perhaps have wrapper routines and crib sheets named after kompsat, that redirect to the existing sosmag routines.

**Comment by jameswilburlewis** [comment]
We should probably also remove the SOSMAG tab from the GUI "Load Data" panel, at least until IDL releases a version that lets us connect to the server.

On the online summary plots, we should have a plot key that calls out the specific SPEDAS and PySPEDAS load routines and crib sheets to use with this data set.   With this in place, we probably don't need to worry about renaming the routines.

**Comment by nickssl** [comment]
In SPEDAS, we have added a readme text file that explains the relationship. SOSMAG is the magnetometer on GEO-KOMPSAT-2A satellite, and the data is on the ESA server that requires authentication. We also include the following links for more information:

https://swe.ssa.esa.int/sosmag
https://link.springer.com/article/10.1007/s11214-020-00742-2

We have also added these two links on the pyspedas readme.md page in the sosmag folder.
